### PR TITLE
Added Palace Overdrive audiobooks integration module

### PR DESCRIPTION
**Pull request by nunommts**
**What's this do?**
This PR imports the Palace module to integrate Overdrive audiobooks. This also updates the expected sha-256 value for the secrets.conf file.

**Why are we doing this? (w/ JIRA link if applicable)**
The audiobooks from Overdrive weren't being correctly downloaded and parsed, as reported [here](https://www.notion.so/lyrasis/Implement-support-for-OverDrive-Audiobooks-in-Android-3b9545183e2342e1a45f89e4a453e010). When opening an Overdrive audiobook, we would get [a downloading error](https://user-images.githubusercontent.com/79104027/149822730-3c778d71-8c42-4d00-bb3f-a2152fec55d3.png)
that was caused by [an incorrect manifest parsing](https://user-images.githubusercontent.com/79104027/149822787-2cf5ed9c-5612-413e-83e6-dd2ac0149c6f.png). This way, it's now possible to listen to Overdrive audiobooks.

**How should this be tested? / Do these changes have associated tests?**
Open the Palace app
Go to settings and enable "Debug Options" by tapping 7 times in the commit version text
Enable "Show non-production libraries" option inside "Testing Features"
Go to the libraries page and add the "Overdrive Integration Test Library" library
Log in to the library
Go to catalog and open any audiobook
Press "Get" and confirm [it's being correctly downloaded](https://user-images.githubusercontent.com/79104027/149823102-cb0ccadb-11fa-47b0-8dff-9c416d904063.png)
Press "Listen" and confirm [it's being correctly played](https://user-images.githubusercontent.com/79104027/149823126-3414d219-dc10-40e5-b1e8-56929df47560.png)

**Dependencies for merging? Releasing to production?**
None

**Have you updated the changelog?**
Yes

**Has the application documentation been updated for these changes?**
No

**Did someone actually run this code to verify it works?**
Tested by @nunommts

### CODE REVIEW

File: simplified-app-palace/build.gradle [hunk 1]
```diff
@@ -9,7 +9,7 @@ def requiredFiles = [:]
 requiredFiles["ReaderClientCert.sig"] =
   "b064e68b96e258e42fe1ca66ae3fc4863dd802c46585462220907ed291e1217d"
 requiredFiles["secrets.conf"] =
-  "d7fcc3c93c0baa881b83683654d27796a3f816e0d77629f4c114493e6a53a743"
+  "756e525becd6f266e8e063abdb8bd03dfca352f0551b33756e21ecb826dfd01b"
```
Comment (ray-lee) [inline simplified-app-palace/build.gradle:13]: @nunommts Is there a PR to mobile-certificates that goes with this?

Comment (nunommts) [reply simplified-app-palace/build.gradle:13]: @ray-lee No there wasn't one, sorry. Here it is now => https://github.com/ThePalaceProject/mobile-certificates/pull/16

**Review by ray-lee — APPROVED** [review]
Looks good!

File: simplified-app-palace/build.gradle (hunk 1 above)
Comment (ray-lee) [reply simplified-app-palace/build.gradle:13]: Hmm, it looks like that isn't the right hash. I'll push an update to fix it.

**Pull request merged**